### ISSUE TITLE
Feature Request: Ability to hide/delete "FBReader: an introduction" in default library view

### ISSUE THREAD

**Issue by dvt32** [geometer/FBReaderJ#395]
It's really rather unnecessary to have to see it all the time. It's distracting and a burden. Apparently you can hide it when using Bookshelf View, but it's not possible to do so in the default library view.

**Comment by kelemvor33** [comment]
Yes please.  This pisses me off.  Like I need to see a Readme file EVERY time I open up the app.

**Comment by ilzoff** [comment]
+1

**Comment by lukash** [comment]
Yes! I don't want it there, after years of using FBReader (great app by the way :)). This should be dead simple to implement.
